worker-batch/src/main/java/com/hpe/caf/worker/batch/BatchWorkerPublisher.java

```diff
@@ -15,112 +15,38 @@
  */
 package com.hpe.caf.worker.batch;
 
-import com.google.common.cache.LoadingCache;
-import com.hpe.caf.api.Codec;
-import com.hpe.caf.api.CodecException;
-import com.hpe.caf.api.worker.TaskMessage;
-import com.hpe.caf.api.worker.TrackingInfo;
-import com.rabbitmq.client.Channel;
-import com.rabbitmq.client.MessageProperties;
+import com.hpe.caf.api.worker.TaskStatus;
+import com.hpe.caf.api.worker.WorkerResponse;
+import com.hpe.caf.api.worker.WorkerTaskData;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.util.AbstractMap;
-import java.util.Map;
-import java.util.concurrent.ExecutionException;
 
 /**
  * Object that publishes task messages from a buffer.
  */
 public class BatchWorkerPublisher {
```

Comment:
You might want to consider whether this class is really still required (or could you just have passed around the `WorkerTaskData` instead).